pylua/test/test_websocket.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+import asyncio
+from gc import collect
+from pylua.network import send_get_echo_message
+from pylua.network import run_websocket_server
+from lupa import LuaRuntime
+
+class TestWebSocket(object):
+    def __init__(self):
+        self._lua_runtime = None
+        self._loop = None
+        self._old_loop = None
+
+    def setup(self):
+        self._lua_runtime = LuaRuntime()
+        self._loop = asyncio.new_event_loop()
+        self._old_loop = asyncio.get_event_loop()
+        asyncio.set_event_loop(self._loop)
+        self._lua_runtime.globals()['run_server'] = self._prepare_server
+        self._lua_runtime.globals()['send_message'] = send_get_echo_message
+
+    def teardown(self):
+        asyncio.set_event_loop(self._old_loop)
+        self._lua_runtime = None
+        self._loop = None
+        self._old_loop = None
+        collect()
+
+    def _prepare_server(self, address, port): #pylint: disable=no-self-use
```

Comment:
@oherasym, make this static method(@staticmethod decorator) or extract as free function. There is no need in suppression.